src/Slider/index.scss

```diff
@@ -0,0 +1,39 @@
+.nds-slider {
+  position: relative;
+  width: 300px;
```

Comment:
The rail width is bounded by this `300px` right? Or is this the initial width?
Would it be possible to make this width auto or `100%`?

Reply:
ah yes good catch